tests/test_t06Ganttreportview.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Selenium IDE
```

Comment:
Is there a friendly description of what each of these tests does?  Nothing to hold up this review to change now, but .py files require unpicking to see what is going on, and a clear plain english description would be nice.  This test after all doesn't seem to do much at all - I'm not sure what it is meant to achieve.

Reply:
See tests/details.json for descriptions of the tests.